MMM-COVID19.js

```diff
@@ -24,12 +24,17 @@ Module.register("MMM-COVID19", {
     headerRowClass: "small", // small, medium or big
     infoRowClass: "big", // small, medium or big
     updateInterval: 300000, // update interval in milliseconds
-    fadeSpeed: 4000
+    fadeSpeed: 4000, 
+	timeFormat:"D.M. HH.mm"
```

Comment:
@hildwin as default, please use `timeFormat: "MMMM Do YYYY, h:mm:ss a"`

You have to also fix the indentation here